### PR TITLE
Ashishonce/integration tests

### PR DESCRIPTION
**Pull request by ashishonce**
added integration tests for ACI and AKS deploy

### CODE REVIEW

File: .cloud/.azure/aci_deploy.json [hunk 1]
```diff
@@ -26,4 +26,13 @@
...
     "dns_name_label": "<your-aci-dns-name-label>"
+}
+
+  
+{
+    "name": "mywebservice",
+    "inference_source_directory": "src/deploy/",
+    "inference_entry_script": "score.py",
+    "conda_file": "environment.yml",
+    "test_enabled": true
```
Comment (marvinbuss) [inline .cloud/.azure/aci_deploy.json:37]: I think something went wrong here. Please remove the last part, if it is not required.

File: .github/workflows/integration.yml [hunk 2]
```diff
@@ -7,10 +7,31 @@ jobs:
...
+        uses: ./
+        with:
+            azure_credentials: ${{ secrets.AZURE_CREDENTIALS }}
+            model_name:  mymodel
+            model_version: 1
+            parameters_file: "test/test_deploy.json"
+
+      # Deploy model in Azure Machine Learning to AKS
+      - name: Deploy model
+        id: aml_deploy
```
Comment (marvinbuss) [inline .github/workflows/integration.yml:31]: Please change name and ID of the second task. Also make the workflow run on PR and push.

File: .cloud/.azure/test/test_deploy.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+    "name": "mywebservice",
+    "inference_source_directory": "tests/deploy/",
+    "inference_entry_script": "score.py",
+    "conda_file": "environment.yml",
+    "test_enabled": false,
+    "delete_service_after_deployment": true
+}
```
Comment (marvinbuss) [inline .cloud/.azure/test/test_deploy.json:8]: Please rename file to test_aci_deploy.json

File: .cloud/.azure/test/test_aks_deploy.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+    "name": "dev-aks",
+    "deployment_compute_target": "devAKS",
```
Comment (marvinbuss) [inline .cloud/.azure/test/test_aks_deploy.json:3]: I think this compute target (aks cluster name) does not exist here. Use the same one as in aml-run.

Comment (ashishonce) [reply .cloud/.azure/test/test_aks_deploy.json:3]: you are right I will correct it

**Comment by marvinbuss** [comment]
The error indicates that we should update the pattern for the service name.

**Pull request merged**